test/infoSpec.js

```diff
@@ -1,98 +0,0 @@
-'use strict'
-
-const assert = require('chai').assert
```

Comment:
all of those tests still exist, they've just been moved to the payment channel framework repo